src/main/java/net/snowflake/ingest/utils/HttpUtil.java

```diff
@@ -44,6 +49,18 @@ public class HttpUtil {
 
   private static CloseableHttpClient httpClient;
 
+  private static final int DEFAULT_CONNECTION_TIMEOUT_MINUTES = 1;
+  private static final int DEFAULT_HTTP_CLIENT_SOCKET_TIMEOUT_MINUTES = 5;
+
+  private static final int DEFAULT_MAX_CONNECTIONS_PER_ROUTE = 100;
+  private static final int DEFAULT_MAX_CONNECTIONS = 100;
+
+  private static final int DEFAULT_TTL_CONNECTION_MINUTES = 2;
+
+  private static final long MONITOR_THREAD_INTERVAL_MS = TimeUnit.SECONDS.toMillis(5);
+
+  private static final int DEFAULT_IDLE_CONNECTION_TIMEOUT_SECONDS = 30;
```

Comment:
Could you explain why we need to override all these values? I thought we only need DEFAULT_MAX_CONNECTIONS_PER_ROUTE and DEFAULT_MAX_CONNECTIONS? Also, is it make more sense to pick a lower DEFAULT_MAX_CONNECTIONS_PER_ROUTE number?

Reply:
I dont have a strong reasoning behind this numbers, except for the fact that I remember we had 100 as max partitions in topic ingesting data to snowflake.

jdbc has this value default to 300 max connections. 

From description: We already have this pushed in master, but I wanted to get this in release branch. 
Adding commit https://github.com/snowflakedb/snowflake-ingest-java/pull/104 on top of 0.10.6 release too.

DEFAULT_IDLE_CONNECTION_TIMEOUT_SECONDS = 30 seconds comes from https://hc.apache.org/httpcomponents-client-4.5.x/current/httpclient/apidocs/org/apache/http/impl/conn/PoolingHttpClientConnectionManager.html

DEFAULT_TTL_CONNECTION_MINUTES = 2 comes from the fact that 2 minutes seems a reasonable time to exhaust an active connection and ask for the new connection from pool. https://stackoverflow.com/questions/39644479/poolingnhttpclientconnectionmanager-what-is-timetolive-attribute-for

Reply:
ok, so seems like we're trying to match what JDBC is doing? If yes, then I would suggest to remove DEFAULT_TTL_CONNECTION_MINUTES, since by default, we're NOT removing it from the connection pool in JDBC (value set to -1)

Reply:
Will revert it to `-1`.